Mathlib/Order/Interval/Finset/Basic.lean

```diff
@@ -237,6 +237,10 @@ theorem Icc_ssubset_Icc_right (hI : a₂ ≤ b₂) (ha : a₂ ≤ a₁) (hb : b
   rw [← coe_ssubset, coe_Icc, coe_Icc]
   exact Set.Icc_ssubset_Icc_right hI ha hb
 
+theorem Ioc_disjoint_Ioc_of_le {d : α} (hbc : b ≤ c) : Disjoint (Ioc a b) (Ioc c d) :=
```

Comment:
This lemma doesn't exist for `Set`, and instead we have `Ioc_disjoint_Ioc_same`

Reply:
Note: I am not saying that `Ioc_disjoint_Ioc_of_le` is worse than `Ioc_disjoint_Ioc_same`. In fact I think it's better. However, sets and finsets should *definitely* have the same lemmas